app/src/main/java/com/eveningoutpost/dexdrip/Models/InsulinInjection.java

```diff
@@ -38,4 +38,5 @@ public boolean isBasal() {
         return getProfile().getMaxEffect() > 1000;
     }
 
+    public double getUnits() { return units; }
```

Comment:
This is not needed as there is a `@Getter` lombok notation above

Reply:
its interesting - Android Studio's syntax inspection reported an error
![Error](https://user-images.githubusercontent.com/51212163/69128879-795d5080-0aad-11ea-8640-d08a7629fc43.jpg)
but when i compile the code i didn't get any....

So just decline the add or leave it as it is to make android studio happy

Reply:
Just enable the lombok plugin in android studio